lib/core/timers/timer-api.ts

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+import { Timeout } from './timer';
+
+let binding = require('../../binding');
+
+export function setImmediate(func: (...args: any[]) => void, ...args: any[]): Timeout {
+    let timeout = new Timeout(func, 0, 0, args);
+    binding.setImmediate(timeout);
+    return timeout;
+}
+
+export function clearImmediate(immediate: Timeout): void {
+    immediate._active = false;
+}
+
+export function setTimeout(func: (...args: any[]) => void, after: number, ...args: any[]): Timeout {
+    let timeout = new Timeout(func, after, 0, args);
+    binding.setTimeoutInterval(timeout);
```

Comment:
use a simpler name than `setTimeoutInterval`. eg `setTimer`

Reply:
Sounds good, change it to setTimers